packages/site/src/content/AnimatedPresence/index.tsx

```diff
@@ -1,26 +1,20 @@
-import {
-  AnimatedPresence as AnimatedPresenceRoot,
-  Button,
-  Content,
-  Flex,
-  Text,
-} from "@jobber/components";
 import AnimatedPresenceContent from "@atlantis/docs/components/AnimatedPresence/AnimatedPresence.stories.mdx";
-import { PropsWithChildren, useState } from "react";
 import Props from "./AnimatedPresence.props.json";
 import { ContentExport } from "../../types/content";
 import { getStorybookUrl } from "../../layout/getStorybookUrl";
 
-export const AnimatedPresence = (props: PropsWithChildren) => {
-  const [switched, setSwitched] = useState(false);
-
+export default {
+  content: () => <AnimatedPresenceContent />,
+  props: Props,
+  component: {
+    element: `const [switched, setSwitched] = useState(false);
   return (
     <Content>
       <Button
         label={switched ? "Switched" : "Initial"}
         onClick={() => setSwitched(!switched)}
       />
-      <AnimatedPresenceRoot {...props}>
```

Comment:
We no longer have to do this Root pattern. We can just use the components as-is.